ansible/ansible.cfg

```diff
@@ -0,0 +1,7 @@
+[defaults]
+private_key_file=~/Downloads/snapcheck-non-prod-server.pem
```

Comment:
not a great location; since it's based on your MAC 
change it to be universal for systems